amplify/backend/function/expressServer/src/routes/auth.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const router = express.Router();
+
+const { resetPassword, recoverPassword } = require('../controllers/auth');
+
+const { check } = require('express-validator');
+
+//Password RESET
+router.post(
+  '/recover',
+  [check('email').isEmail().withMessage('Enter a valid email address')],
```

Comment:
This doesn't seem to work, I dont know this syntax but I cant get this mesage. 
![image](https://user-images.githubusercontent.com/61060096/201648875-c2361987-65a6-4269-8e85-fe003f45b21d.png)
![image](https://user-images.githubusercontent.com/61060096/201648917-1b565022-0618-4ea6-aac9-b9b579fd63f6.png)
![image](https://user-images.githubusercontent.com/61060096/201649043-fb7edc53-52ed-4e9c-9efa-b0e3272b7409.png)